src/index.tsx

```diff
@@ -8,7 +8,9 @@ import AuthProvider from "./components/Auth/AuthProvider";
 import { Provider } from "react-redux";
 import { store } from "./state/store";
 
-const root = ReactDOM.createRoot(document.getElementById("root"));
+const root = ReactDOM.createRoot(
+  document.getElementById("root") as HTMLElement
```

Comment:
Out of curiosity do you know why typescript didn't complain about this before?

Reply:
Originally this was still a `.js` file instead of a `.tsx`. Once I changed the filetype to tsx it needed to know the type

Reply:
Oh ok makes sense, thanks!